File: ADL-332 BigData Analytics lab/Palindrome.r
is_palindrome <- function(x) {
  reversed <- paste(rev(strsplit(x, "")[[1]]), collapse = "")
  if (x == reversed) {
    return("Palindrome")
  } else {
    return("Not Palindrome")
  }
}
# Test
is_palindrome("madam")
is_palindrome("racecar")
is_palindrome("hello")

File: ADL-332 BigData Analytics lab/Palindrome.py
s = input()
if s == s[::-1]:
    print("Palindrome")
else:
    print("Not Palindrome")



/// Another way 
s=input()
rev=""
for i in s:
  rev=i+rev
if rev==s:
  print("IS palindrome")
else:
  print("Not a palindrome")
